wolfcrypt/src/asn.c

```diff
@@ -26137,6 +26137,10 @@ int wc_EccPublicKeyDecode(const byte* input, word32* inOutIdx,
                     == 0 && tag == ASN_BIT_STRING) {
                 len = 0;
                 ret = GetASNHeader(input, ASN_BIT_STRING, inOutIdx, &len, inSz);
+                if (ret > 0) {
+                    len = ret;
```

Comment:
No need to set len.

Reply:
Right. I got fixated on fixing ret that I didn't notice that. Thanks.